adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -121,14 +130,11 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
     if (args.recalibrateBaseQualities) {
       log.info("Recalibrating base qualities")
       val knownSnps: SnpTable = createKnownSnpsTable(sc)
-      val stringency = if (args.strictBQSR) {
```

Comment:
Yea, across many things that `transform` does, it's reasonable to let the user say "be {strict,lenient,silent} with any applicable validation checks", and that that should happen via one flag that is not explicitly namespaced to any one task that the user may already separately be telling `transform` to do (e.g. BQSR).